src/app/forms/suggestion-box/helper/terra-suggestion-box.helper.ts

```diff
@@ -0,0 +1,33 @@
+import { isNullOrUndefined } from 'util';
+import { TerraSuggestionBoxValueInterface } from '../data/terra-suggestion-box.interface';
+
+export class TerraSuggestionBoxHelper
+{
+
+    public static getSuggestionBoxEntryForCaption(caption:string,
+                                                  terraSuggestionBoxValues:Array<TerraSuggestionBoxValueInterface>,
+                                                  fallbackValue:TerraSuggestionBoxValueInterface):TerraSuggestionBoxValueInterface
+    {
+        if(isNullOrUndefined(terraSuggestionBoxValues))
+        {
+            return fallbackValue;
```

Comment:
Was ist wenn `fallbackValue == null` ist?
Mach die Angabe doch optional und generiere dir in diesem Fall standardmäßig den Fallback anhand der Caption.

Reply:
Die frage war eher so gemeint,  ob das Absicht ist oder ob es immer einen Fallback geben soll?

Reply:
Verstehe ich nicht, ja das ist Absicht